submodules/LIB/hardware/fpga/Makefile

```diff
@@ -36,6 +36,7 @@ include $(FPGA_TOOL)/build.mk
 VHDR += $(wildcard ../src/*.vh) $(wildcard ./src/*.vh)
 VSRC += $(wildcard ../src/*.v) $(wildcard ./src/*.v)
 VSRC += $(wildcard ../common_src/*.v)
+VSRC+=$(wildcard $(REMOTE_FPGA_DIR)/src/*.v)
```

Comment:
I don't understand why this line was added.
This is the Makefile inside the FPGA directory, so the working directory for that Makefile will be inside the `/hardware/fpga/` path.

The `$(REMOTE_FPGA_DIR)` variable contains something like `<username>/<project_name>/hardware/fpga`.

So, by adding the line:
`$(wildcard $(REMOTE_FPGA_DIR)/src/*.v)`
We are trying to find Verilog sources in the following directory:
`/hardware/fpga/<username>/<project_name>/hardware/fpga/src/*.v`
And, as far as I know, this directory does not exist.